src/meta/src/hummock/compaction/tier_compaction_picker.rs

```diff
@@ -110,7 +112,10 @@ impl TierCompactionPicker {
                 max_level_size * self.config.level0_tier_compact_file_number / 2 > compaction_bytes;
 
             // do not pick a compact task with large write amplification.
```

Comment:
We need to add some new comments to description the reason (avoid not triggering when sst is too large e.g.)